### ISSUE TITLE
The Run button never return from compiling

### ISSUE THREAD

**Issue by suveergarg** [janjongboom/mbed-simulator#42]
Hi, I am running an UBUNTU 16.04.4LTS. When I open the simulator on my Firefox Browser, the run button, gets stuck and never returns from compiling. At the same time when I rebuild demos using bash build-demos.sh, it never exits. Please help

**Comment by janjongboom** [comment]
Any logs? E.g. Do you see the compilation step starting? Do you have an ad blocker that might be interfering?

**Comment by suveergarg** [comment]
No, there is no add-blocker. I run it on a clean virtual machine. It works fine for some-time before the run button gives up. Even running bash build-demos.sh does not work from the terminal. On restarting my VM, things start working again before I run into the same issue. The button gets stuck on compiling. Refreshing the page does not help. 

Thanks
